### DISCUSSION TITLE
Check update only on windows startup

### DISCUSSION THREAD

**Discussion by ghost** [d2phap/ImageGlass#801] [General] [answered]
Hello,
There is a way to check if there is update on windows startup and not on the software startup.
Because i'm not using this often every time i want to use it, i have to do an update it's very annoying.

Thanks For the help.

**Comment by d2phap** [comment]
Hi @aelisya 
IG is checking for update every 3 days.
You can disable this by going to Settings > General > Others, uncheck the option: Check for update automatically.

**Comment by ghost** [comment]
yes but there is a way (even if i have to create a script myself, to launch the search of update on windows startup and not on the software startup ?).
Because i don't use often the software so i don't see the update (even if there is one for more than 3 day)

**Comment by d2phap** [comment, marked as answer]
Then you can still disable Auto-update and run this command to check for update.
This command will only show the UI if there is an update.

```
/path/to/igcmd.exe igautoupdate
```


https://imageglass.org/docs/command-line-utilities

**Reply by ghost** [reply]
thanks i haven't see it (sorry for the time loss)
